samples/boards/96b_carbon/multi_thread_blinky/README.rst

```diff
@@ -0,0 +1,37 @@
+.. _96b_carbon_multi_thread_blinky:
+
+96Boards Carbon Basic Multi Thread Example
+##########################################
+
+Overview
+********
+
+This example demonstrates spawning of multiple threads using K_THREAD_DEFINE.
+
+The example works by spawing three threads each to controll a separate LED 
+on the 96Boards Carbon Board. Each of these LEDs (USR1, USR2 and BT) have 
+their individual loop control and timing logic defind inside separate functions.
+
+blink1() controls the USR1 LED that has a 100ms sleep cycle
+blink2() controls the USR2 LED that has a 1000ms sleep cycle
+blink3() controls the BT LED that increases sleep by 100ms every cycle
+	 upto 600ms and then resets to 100ms
+
+
+Each thread is then spawned during compile time using K_THREAD_DEFINE
```

Comment:
I think technically, the thread can't be "spawned during compile time", so maybe change to:

     Each thread is then defined at compile time using K_THREAD_DEFINE.